articles/introduction-to-lxd-container-manager/index.md

```diff
@@ -0,0 +1,38 @@
+### Introduction
+LXD is a highly effective container manager. It provides an individual experience same as that offered by virtual machines. LXD systems are unique because they operate with Linux containers. This technology was introduced by Canonical Ltd alongside other contributors. LXD is supported by numerous Linux distributions. One of the significant pillars of LXD is a powerful REST API. 
+
+### Goal
+This article discusses the key features of LXD, the benefits, as well as its releases. 
+
+### Understanding what LXD entails
+LXD is a lightweight container manager which operates in the same way as a virtual machine. It does not require a Linux kernel and CPU hardware virtualization. LXD containers are more advantageous as compared to virtual machines. This is because the server can support more processes than virtual containers. LXD offers a hypervisor that helps to manage the life cycle of containers effectively. Furthermore, LXD containers are easy to control since all operations are done via a REST API. This means that accessing a remote server or the local host is seamless.
+
+### Key features of LXD
+LXD Containers are image-based. This is effectively achieved by a variety of Linux distributions worldwide. Linux containers possess great storage management. This is ideal for its multiple storage, pools, and volumes. Another feature is the device passthrough. Some of these components include block devices, Unix character paths, discs, NICs, GPU, and USB. Linux containers perform well in network management. The key activities in network management include configuration, creation of a network bridge, and the use of cross-host tunnels.
+
+LXD containers are quite secure. This is quite advantageous, especially in the modern world where cyber threats are on the rise. LXD’s built-in security features include resource restrictions and unprivileged containers. Resource restrictions limit how users access containers. This means that unauthenticated individuals cannot access the core system. Unprivileged containers ensure that users only have control over containers that they own. LXD containers are scalable. They can, therefore, accommodate many users and computations nodes. Another feature of LXD containers is that they are intuitive and simple. They utilize a powerful and clear API. It is easy for inexperienced users to make maximum use of these systems. The command line is also simple to use. Support for image transfer and cross host containers is another great feature of LXD. These containers support live migration. Lastly, LXD containers possess advanced resource control. This is supported by kernel resources, disk usage, memory, and CPU.     
```

Comment:
Break this down paragraph into 2 or more paras.

One para must convey one idea.